src/pages/documentation/media-for-edge-network/tabs/api-reference.md

```diff
@@ -0,0 +1,2117 @@
+      public static final String DAY_PART = "a.media.dayPart";
+      public static final String FEED = "a.media.feed";
+      public static final String STREAM_FORMAT = "a.media.format";
+  }
+
+}
+```
+
+<Variant platform="ios" api="standard-video-constants" repeat="6"/>
+
+```swift
+public class MediaConstants: NSObject {
+  @objc(AEPEdgeMediaVideoMetadataKeys)
+  public class VideoMetadataKeys: NSObject {
+        public static let SHOW = "a.media.show"
+        public static let SEASON = "a.media.season"
+        public static let EPISODE = "a.media.episode"
+        public static let ASSET_ID = "a.media.asset"
+        public static let GENRE = "a.media.genre"
+        public static let FIRST_AIR_DATE = "a.media.airDate"
+        public static let FIRST_DIGITAL_DATE = "a.media.digitalDate"
+        public static let RATING = "a.media.rating"
+        public static let ORIGINATOR = "a.media.originator"
+        public static let NETWORK = "a.media.network"
+        public static let SHOW_TYPE = "a.media.type"
+        public static let AD_LOAD = "a.media.adLoad"
+        public static let MVPD = "a.media.pass.mvpd"
+        public static let AUTHORIZED = "a.media.pass.auth"
+        public static let DAY_PART = "a.media.dayPart"
+        public static let FEED = "a.media.feed"
+        public static let STREAM_FORMAT = "a.media.format"
+    }
+}
+```
+
+**Example**
+
+#### Swift
+
+```swift
+var mediaObject = Media.createMediaObjectWith(name: "video-name", id: "videoId", length: 60, streamType: MediaConstants.StreamType.VOD, mediaType: MediaType.Video)
+
+var videoMetadata: [String: String] = [:]
+// Standard Video Metadata
+videoMetadata[MediaConstants.VideoMetadataKeys.SHOW] = "Sample show"
+videoMetadata[MediaConstants.VideoMetadataKeys.SEASON] = "Sample season"
+
+tracker.trackSessionStart(info: mediaObject, metadata: videoMetadata)
+```
+
+#### Objective-C
+
+```objc
+NSDictionary *mediaObject = [AEPMobileEdgeMedia createMediaObjectWith:@"video-name" id:@"video-id" length:60 streamType:AEPEdgeMediaStreamType.VOD mediaType:AEPEdgeMediaTypeVideo];
+
+NSMutableDictionary *videoMetadata = [[NSMutableDictionary alloc] init];
+// Standard Video Metadata
+[videoMetadata setObject:@"Sample show" forKey:AEPEdgeMediaVideoMetadataKeys.SHOW];
+[videoMetadata setObject:@"Sample Season" forKey:AEPEdgeMediaVideoMetadataKeys.SEASON];
+
+[_tracker trackSessionStart:mediaObject metadata:videoMetadata];
+```
+
+<Variant platform="android" api="standard-audio-constants" repeat="1"/>
+
+```java
+public class MediaConstants {
+
+  public static final class AudioMetadataKeys {
+    public static final String ARTIST = "a.media.artist";
+    public static final String ALBUM = "a.media.album";
+    public static final String LABEL = "a.media.label";
+    public static final String AUTHOR = "a.media.author";
+    public static final String STATION = "a.media.station";
+    public static final String PUBLISHER = "a.media.publisher";
+  }
```

Comment:
Can you double check these values? I believe the "a.media." prefix is dropped in Edge Media.